### PR TITLE
docs(install): Add PowerShell Web Installer

### PR DESCRIPTION
**Pull request by jpawlowski**
#### Description

This will add some information about an alternative installation method for PowerShell.
Also see discussion here: https://github.com/ryanoasis/nerd-fonts/discussions/1697

#### Requirements / Checklist

- [X] Read the [Contributing Guidelines](https://github.com/ryanoasis/nerd-fonts/blob/-/contributing.md)
- [X] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan.
      Issue number where discussion took place: #xxx
- [X] If this contains a font/glyph add its origin as background info below (e.g. URL)
- [X] Verified the license of any newly added font, glyph, or glyph set. License is: xxx

#### What does this Pull Request (PR) do?

Update `README.md` with info about PowerShell web install option.

#### How should this be manually tested?

n/a

#### Any background context you can provide?

n/a

#### What are the relevant tickets (if any)?

https://github.com/ryanoasis/nerd-fonts/discussions/1697

#### Screenshots (if appropriate or helpful)

PowerShell Nerd Fonts Installer

### CODE REVIEW

**Comment by jpawlowski** [comment]
@all-contributors please add @jpawlowski for doc

File: readme.md [hunk 1]
```diff
@@ -244,7 +246,55 @@ scoop install Hack-NF
...
 Some special packages are [in AUR](https://aur.archlinux.org/packages?K=nerd-fonts-&outdated=off).
 
-### `Option 5: Ad Hoc Curl Download`
+### `Option 5: PowerShell Web Installer`
+
+> Best option for **interactive setup guidance** or **automating** installations through **PowerShell scripts**.
+
+_Note_:
+  - **Requires PowerShell 7+** to be installed on either Windows, macOS, or Linux.
+  - **Windows PowerShell 5.1** works okay too.
```
Comment (Finii) [inline readme.md:255]: I would appreciate another note that the web installer is a 3rd party tool, together with your repo's link, and that maintenance is outside of the NerdFont project.

**Review by Finii — COMMENTED** [review]
Looks very good, thank you!

I guess `pwsh` is nowadays called `powershell`? I never used powershell I have no clue 😬 (I can hardly believe it is not executed via `pwsh` on linuxerian OSes).

And this certainly renders it useless , 2404 is ages old already. Whatever, that does not matter for this PR.

Screenshot 2024-09-01 at 16 13 57

**Comment by Finii** [comment]
Hmm, not supporting Ubuntu 24.04 makes no sense.
And/but they support MacOS 14.6.1 (latest latest) via Homebrew? This is strange, why is it not community-supported on Ubuntu?

```console
fini@Air scripts % uname -a
Darwin Air 23.6.0 Darwin Kernel Version 23.6.0: Mon Jul 29 21:16:46 PDT 2024; root:xnu-10063.141.2~1/RELEASE_ARM64_T8112 arm64
fini@Air scripts % brew info powershell
==> powershell: 7.4.2
https://github.com/PowerShell/PowerShell
Not installed
From: https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/p/powershell.rb
==> Name
PowerShell
==> Description
Command-line shell and scripting language
==> Artifacts
powershell-7.4.2-osx-arm64.pkg (Pkg)
==> Caveats
To use Homebrew in PowerShell, run the following in a PowerShell session:
  New-Item -Path (Split-Path -Parent -Path $PROFILE.CurrentUserAllHosts) -ItemType Directory -Force
  Add-Content -Path $PROFILE.CurrentUserAllHosts -Value '$(/opt/homebrew/bin/brew shellenv) | Invoke-Expression'

==> Analytics
install: 11,523 (30 days), 37,265 (90 days), 102,246 (365 days)
fini@Air scripts %
```

File: readme.md (hunk 1 above)
Comment (jpawlowski) [reply readme.md]: Done in b622bc3f0ab0cff26bdbdf0ed3df4d421ac22e9d

**Comment by jpawlowski** [comment]
> I guess `pwsh` is nowadays called `powershell`? I never used powershell I have no clue 😬 (I can hardly believe it is not executed via `pwsh` on linuxerian OSes).

Indeed, PowerShell 7 was initially known as "PowerShell Core" following its complete overhaul in version 6. It's distinct from Windows PowerShell 5.1, which is reflected in its binary file name `pwsh.exe` for Windows, allowing it to run alongside the older version. Similarly, Unix systems use the `pwsh` name for their binaries.

Despite these changes, for backward compatibility, documentation and markdown language tags often simply refer to it as `powershell`. For instance, the [Microsoft documentation on including code](https://learn.microsoft.com/en-us/contribute/content/code-in-docs#supported-languages) adheres to this convention. GitHub's syntax highlighter, powered by [`linguist`](https://github.com/github-linguist/linguist/blob/master/lib/linguist/languages.yml), recognizes `pwsh` as a synonym for `powershell`, which might have led to the oversight in updating your documentation. Apologies for any confusion caused.

> And this certainly renders it useless , 2404 is ages old already. Whatever, that does not matter for this PR.

Regarding Ubuntu releases, the designation of the latest LTS version is finalized with the first `.1` point release, which happened on [August 29th, 2024](https://lists.ubuntu.com/archives/ubuntu-announce/2024-August/000304.html). Firms like Microsoft usually await this milestone before incorporating the new LTS version into their lineup of supported OS. Technically, Microsoft has prepared much of the necessary infrastructure. I've also pitched in with a small update for the Dev Container features, so we can expect full support to roll out soon. :-)

> Hmm, in comparsion the new install method takes quite a lot of space; maybe hide some of the sub-points by just linking to the documentation on your installers project and list only the one or two major things explicitly.

I've taken this into account in the latest update. Could you please review it again?

**Comment by Finii** [comment]
Thanks again :+1: :green_heart:

**Pull request merged**